### DISCUSSION TITLE
Add actions to go to first/last match in a buffer search

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#26085] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

The title says it all...

**Comment by notpeter** [comment]
Is this supported in other editors?
Do any of them map a key for it?

**Comment by failable** [comment]
In Emacs one can use `iedit` to go to the first and last match by using, the keymap is only available when search is active.

```elisp
    (define-key map (kbd "M-<") 'iedit-goto-first-occurrence)
    (define-key map (kbd "M->") 'iedit-goto-last-occurrence)
```
